assets/script.js

```diff
@@ -194,40 +209,63 @@ fetch(searchNinjaUrl,
     console.log("search input did not have output. try something else");
     return;  //ends function early for bad search input.
   }
-  console.log(data); 
-  sportResult = data[0];
-  sportCalories = sportResult.calories_per_hour;
-  console.log(sportResult);
-  console.log(sportResult.name);
-  console.log(sportResult.calories_per_hour);
-  console.log(sportCalories);
+  sportInfoCurrent.push(data)
+  // console.log(data); 
+  // sportResult = data[0];
+  // sportCalories = sportResult.calories_per_hour;
+  // console.log(sportResult);
+  // console.log(sportResult.name);
+  // console.log(sportResult.calories_per_hour);
+  // console.log(sportCalories);
 })
 .catch(function (error) {
   console.error(error);
   notFound.textContent = "searchNinjaUrl_error";
 });
+console.log(i)
+}
+if (i >= sportSet.length){
 return;
 }
+}
 
 async function fetchActivities(calories) {
```

Comment:
you can take this example out since you got a working fetch function for activities now.

Reply:
True. Forgot it was there. Will remove.